### DISCUSSION TITLE
Faction Garden🌼

### DISCUSSION THREAD

**Discussion by Alucard0389** [sunflower-land/sunflower-land#4206] [Ideas]
![image](https://github.com/user-attachments/assets/880fdebc-865e-4ea0-828f-702af9a6812a)

I've been giving some thought to this door and where it might lead.

You may have to complete a longer quest series to get the garden key. 
After you have received it, it opens the gates to the Faction Garden

I have 4 different flowers for each faction

Night Shaders

Necrobloom: A deep purple flower with black tendrils and an aura-like, greenish glowing center. It grows in nighttime, cursed forests and graveyards. This plant has the ability to temporarily raise the dead to aid the Night Shaders in battle.

Sunflorians

Solaraurelia: A golden plant with brilliant, sunny yellow petals that protrude from the center like rays of sunlight. It only grows in strong sunlight and can release magical healing powers that regenerate and strengthen Sunflorians in battle.

Goblins

War Thistle: A tough, spiky plant with dark green leaves and red flowers covered in sharp thorns. The War Thistle grows in rocky and inhospitable areas. Its thorns contain a poison that goblins use to poison their weapons and weaken enemies.

Bumkins

Fieldflower: A hardy, versatile plant with white flowers and bold green leaves. It grows in fields and gardens and is known for its nutritional benefits. The Fieldflower can be used both as a nutritional supplement and to make healing salves, which helps the Bumkins in everyday life and in battle.

🤔

(Faction Garden) Mechanics

1. Establishing the Faction Garden

Location and Access:

    •    Each faction has its dedicated garden area where members can contribute.
    •    Access is granted to all faction members, with special permissions for higher-ranked players to manage the garden.

Planting the Necroflower:

    •    A certain number of seeds are required to start growing a Necroblüte in the faction garden.
    •    Seeds can be obtained through quests, special events, or purchased using faction points.

2. Collective Contributions

Watering and Fertilizing:

    •    Members can contribute by watering and fertilizing the Necroblüte, with each action speeding up the growth process.
    •    Introduce a cooldown period for watering and fertilizing to encourage ongoing participation.

Special Tasks:

    •    Assign daily or weekly tasks such as pest control, weeding, or applying special potions.
    •    Completing tasks grants individual rewards and contributes to the overall growth of the Necroblüte.

3. Growth Phases and Rewards

Growth Phases:

    •    The Necroblüte goes through several growth phases, each requiring different collective efforts (e.g., more watering, specific fertilizers).
    •    Visual indicators show the current growth stage and what is needed to advance to the next phase.

Faction-Wide Buffs and Rewards:

    •    Upon reaching certain growth milestones, the entire faction receives buffs (e.g., increased resource yields, faster crafting times).
    •    Harvesting a fully grown Necroblüte grants unique faction-wide rewards, such as rare items, exclusive cosmetics, or special abilities.

4. Competition and Collaboration

Faction Challenges:

    •    Introduce inter-faction challenges where factions compete to grow the most Necroblüte within a given timeframe.
    •    Rewards for winning factions could include exclusive items, bragging rights, and leaderboard recognition.

5. Visual and Interactive Features

Garden Customization:

    •    Allow factions to customize their garden with decorations, enhancing the aesthetic appeal and reflecting faction identity.
    •    Unlock decorations through achieving milestones or completing special tasks.

Interactive Elements:

    •    Introduce interactive elements such as mini-games related to gardening tasks, increasing engagement and fun.
    •    Real-time notifications to inform faction members of the garden’s status and needed actions.

Implementation Example

Daily Routine:

    1.    Watering: Each member can water the plant once every 12 hours, contributing to its growth.
    2.    Fertilizing: Use faction-earned resources to craft special fertilizers that speed up growth or enhance the plant’s qualities.
    3.    Task Completion: Participate in daily tasks like pest control or applying growth potions to ensure healthy growth.

Growth Milestones:

    •    Sprouting Phase: Initial stage requiring basic watering and fertilizing.
    •    Blooming Phase: Mid-stage needing more complex fertilizers and pest control.
    •    Mature Phase: Final stage where special potions and consistent care ensure a rich harvest.

Rewards:

    •    Faction Buffs: Each milestone reached grants buffs like faster crop growth or increased resource collection.
    •    Unique Items: Harvesting the mature Necroblüte provides rare items and cosmetics for all faction members.

You could add many new mechanics to the garden

I would be interested to hear what the rest think of it or what other ideas we can find together🤟

**Comment by ShinKan42** [comment]
Hopefully combined mech with Blacksmith system.
Garden is so suited for shared resource production like crafting material for the whole faction.

**Reply by Alucard0389** [reply]
Yes maybe you can craft a watering can or something like fertilizer at blacksmith buying materials maybe for marks🤔

**Comment by adamhannigan** [comment]
@Alucard0389 Excellent idea! This area is open for a few possible solutions.

We are hoping to use it as a co-operative planting area. I think the garden/flowers could fit nicely here 🙏
